command/agent/agent_test.go

```diff
@@ -18,7 +18,6 @@ import (
 
```

Comment:
Was running these in parallel breaking something? The tests were passing on this branch for me when I last touched it.

Reply:
no, but it wasn't helping either since `go test` already does package level parallelization. My CPU burned through all the tests in the same time whether the tests were running in parallel or not.

Reply:
Tried it again. Test run time goes from 55sec to 51sec when running tests in parallel. Maybe I should leave this in.